torch/csrc/jit/frontend/schema_type_parser.cpp

```diff
@@ -203,6 +203,7 @@ std::pair<TypePtr, c10::optional<AliasInfo>> SchemaTypeParser::parseType() {
     auto subalias = std::move(p.second);
     L.expect(')');
     value = FutureType::create(subtype);
+    alias_info = parseAliasAnnotation();
```

Comment:
this is not needed as we don't want to parse alias annotation for future, it's special handled.